components/dashboard/src/components/PendingChangesDropdown.tsx

```diff
@@ -4,12 +4,19 @@
  * See License.AGPL.txt in the project root for license information.
  */
 
-import { WorkspaceInstance } from "@gitpod/gitpod-protocol";
+import { WorkspaceInstance, WorkspaceInstanceRepoStatus } from "@gitpod/gitpod-protocol";
 import ContextMenu, { ContextMenuEntry } from "./ContextMenu";
 import CaretDown from "../icons/CaretDown.svg";
+import { useFeatureFlag } from "../data/featureflag-query";
 
 export default function PendingChangesDropdown(props: { workspaceInstance?: WorkspaceInstance }) {
-    const repo = props.workspaceInstance?.status?.repo;
+    const liveGitStatus = useFeatureFlag("supervisor_live_git_status");
+    let repo: WorkspaceInstanceRepoStatus | undefined;
+    if (liveGitStatus) {
+        repo = props.workspaceInstance?.repo;
+    } else {
+        repo = props.workspaceInstance?.status?.repo;
```

Comment:
I think we should have this logic the default no matter what the feature flag is. I.e. 
```
const repo = props.workspaceInstance?.repo || props.workspaceInstance?.status?.repo;
```

Reply:
the feature flag is checked in supervisor already. I think that is the right place and enough.

Reply:
Which one we should use on the client then?

Reply:
> const repo = props.workspaceInstance?.repo || props.workspaceInstance?.status?.repo;

I did it on purpose that we don't misguide us, i.e. right now it is clear that the source is live status. if it does not work we can notice.